app/helpers/application_helper.rb

```diff
@@ -1,4 +1,12 @@
 module ApplicationHelper
+  def page_text_direction
+    I18n.t("shared.language_direction.#{@content_item["locale"]}", default: "ltr")
+  end
+
+  def text_direction_wrapper_class
```

Comment:
Though a class is useful, this could also be a [`dir` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir) on the wrapping element:

```html
<p dir="ltr">This is left to right</p>
```

If the CSS doesn't load, then the browser can still display correctly.

If you wanted to remove the class completely, you can still style the element to make it go right to left:

```css
[dir="rtf"] {
  direction: rtl;
  text-align: start;
}
```

Reply:
Thanks @injms I had just noticed that and pushed up a simplified commit using the `dir` attribute :+1:

Reply:
Super!